frontend/src/components/navbar/NavBar.js

```diff
@@ -34,37 +34,31 @@ const Navbar = ({ navigate }) => {
     contextButton = <div className="nav-item" onClick={logout}>
       <a href="/login">Logout</a>
     </div>
-    // } else if (in login page) {
-    //   contextButton = <div className="nav-item">
-    //                     <a href="/signup">Sign up</a>
-    //                   </div>
-    // } else if (in signup page) {
-    //   contextButton = <div className="nav-item">
-    //                     <a href="/login">Login</a>
-    //                   </div>
-    // }
-  } else {
-    contextButton = <div className="nav-item">
-                       <a href="/login">Login</a>
-                       <a href="/signup">Sign up</a>
-                    </div>
-  }
+    } else if (window.localStorage.getItem("app-route") === "login") {
+      contextButton = <div className="nav-item">
+                        <a href="/signup">Sign up</a>
+                      </div>
+    } else if (window.localStorage.getItem("app-route") === "signup") {
+      contextButton = <div className="nav-item">
+                        <a href="/login">Login</a>
+                      </div>
+    }
   
     return (
       <nav>
-        <div className="nav-bar">
-        <div className="nav-left">
-        <Avatar
-          alt="Remy Sharp"
-          src="https://res.cloudinary.com/dmkipvd8d/image/upload/w_1000,c_fill,ar_1:1,g_auto,r_max,bo_5px_solid_red,b_rgb:262c35/v1684179149/800px-Facebook_logo__28square_29_dgjsjn.png"
-          sx={{ width: 56, height: 56 }}
-        />
+        <div className="navbar">
+          <div className="nav-left">
+            <Avatar
+              alt="Remy Sharp"
+              src="https://res.cloudinary.com/dmkipvd8d/image/upload/w_1000,c_fill,ar_1:1,g_auto,r_max,bo_30px_solid_white,b_rgb:262c35/v1684179149/800px-Facebook_logo__28square_29_dgjsjn.png"
+              sx={{ width: 56, height: 56 }}
+            />
         
-        <h1 className="title">Farcebook</h1>
-        </div>
-          <>
-           { contextButton }
-          </>
+            <h1 h1 className="title">Farcebook</h1>
```

Comment:
duplicate of h1 to remove here - is causing an error message in chrome dev console